config/routes/api_routes.rb

```diff
@@ -11,6 +11,7 @@ def self.extended(router)
         end
 
         resource :info, only: :show, controller: "info", constraints: RouteConstraints::RegisterApiConstraint
+        resources :trainees, only: %i[show create], controller: "trainees", constraints: RouteConstraints::RegisterApiConstraint
```

Comment:
Does this overlap with line 7 above? Can the `create` just be added there instead of this line?